deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/iterator/BertIterator.java

```diff
@@ -299,6 +354,52 @@ public MultiDataSet next(int num) {
         return new Pair<>(outLength, tokenizedSentences);
     }
 
+    private Pair<long[], List<Pair<List<String>, String>>> tokenizePairsMiniBatch(List<Triple<String, String, String>> listPairs) {
```

Comment:
Maybe let's make the return type of this method a pojo instead - a simple private static class.
That should be easier to interpret than 4 layers of generics...

Reply:
Done